mlir/lib/Dialect/SPIRV/Transforms/UpdateVCEPass.cpp

```diff
@@ -95,6 +95,16 @@ static LogicalResult checkAndUpdateCapabilityRequirements(
   return success();
 }
 
+static SetVector<spirv::Capability>
+withImpliedCapabilities(SetVector<spirv::Capability> &caps) {
```

Comment:
```suggestion
withImpliedCapabilities(const SetVector<spirv::Capability> &caps) {
```

This argument doesn't seem to get modified

Reply:
Logic changed not longer applies